tests/script/test_evaluate.py

```diff
@@ -418,16 +418,22 @@ async def test_scriptEvaluate_dedicated_worker(websocket, context_id, html,
             }
         })
 
-    # Wait for worker to be created
-    while True:
```

Comment:
If the event format is changed (eg added a new field), this test failed with timeout instead of providing a meaningful message.

Reply:
I split the assertion into 2 parts:
* Get the event.
* Assert the event.